sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
```

Comment:
Service team is working on a [new version](https://github.com/Azure/azure-sdk-for-net/pull/36803), please update this pr after that pr got merged.

Reply:
Merged now